_posts/news/2022-04-04-announcing-scalajs-1.10.0.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Announcing Scala.js 1.10.0
+category: news
+tags: [releases]
+permalink: /news/2022/04/04/announcing-scalajs-1.10.0/
+---
+
+
+We are pleased to announce the release of Scala.js 1.10.0!
+
+**This releases addresses a security vulnerability in `java.util.UUID.randomUUID()`.**
+We strongly recommend that all users upgrade to Scala.js 1.10.0 as soon as possible.
+Read [the security advisory](https://github.com/scala-js/scala-js/security/advisories) for more details.
```

Comment:
We need to make sure that somewhere, we list the affected Scala.js versions of this (IIUC, basically all of them, but it's still important we make that clear).